config/jobs/image-pushing/k8s-staging-release-test.yaml

```diff
@@ -1,4 +1,34 @@
 periodics:
+- interval: 2h
+  name: ci-release-anago-stage
+  cluster: test-infra-trusted
+  decorate: true
+  extra_refs:
+  - org: kubernetes
+    repo: release
+    base_ref: master
+    path_alias: k8s.io/release
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-release-test/k8s-cloud-builder:latest
```

Comment:
This cannot be permitted to run in the trusted cluster without further discussion with the test-infra team: we do not permit code we do not maintain to run there for security reasons.

/hold

Reply:
Also even if we did, we probably would not be super excited about a `:latest` ...
/lgtm cancel